graphwalker-dsl/src/main/java/org/graphwalker/dsl/antlr/generator/GeneratorLoader.java

```diff
@@ -41,89 +40,92 @@
  */
 public class GeneratorLoader extends Generator_ParserBaseListener {
 
-  StopCondition stopCondition = null;
```

Comment:
The changes reflected by this diff is way more than the actual changes.
It looks to me like it's a formatting issues?